strawberry_django_plus/utils/typing.py

```diff
@@ -12,7 +12,8 @@
 DictTree: TypeAlias = Dict[str, "DictTree"]
 TypeOrSequence: TypeAlias = Union[_T, Sequence[_T]]
 TypeOrIterable: TypeAlias = Union[_T, Iterable[_T]]
-UserType: TypeAlias = Union[AbstractUser, AnonymousUser]
+# deprecated
+UserType: TypeAlias = AbstractBaseUser
```

Comment:
My suggestion was not to deprecate `UserType`, but make it a union of `AbstractBaseUser` and `AnonymousUser`

The reason `UserType` exists is to represent a union between the user and anonymous user, since both doesn't inherit from the sabe base type

Reply:
Having said all of that and looking at the whole PR, we only import `AbstractUser` for typing, meaning that instead of changing `UserType` here or even in any other place, we could import it in a `if TYPE_CHECKING` block, meaning that it will not raise issues when the app is not installed and the typing will still work fine. What do you think?